src/components/slide/selection-frame.tsx

```diff
@@ -221,48 +205,83 @@ export const SelectionFrame: React.FC<Props> = ({ children, treeId }) => {
 
   const isHovered = hoveredElementId === children.props.id;
   const isSelected = editableElementId === children.props.id;
-  const isSelectedAndMarkdown = isSelected && isMdElement(selectedElement);
+
+  const handleMouseDown = useCallback(
+    (event: MouseEvent<HTMLDivElement>) => {
+      if (
+        (event.target as HTMLElement).classList.contains('moveable-control')
+      ) {
+        return;
+      }
+      event.stopPropagation();
+      dispatch(deckSlice.actions.editableElementSelected(children.props.id));
+
+      if (event.detail === 2) {
+        setDoubleClickedElement(true);
+      }
+    },
+    [children.props.id, dispatch]
+  );
+
+  useEffect(() => {
+    if (doubleClickedElement && selectedElement?.id !== children.props.id) {
+      setDoubleClickedElement(false);
+    }
+  }, [children.props.id, doubleClickedElement, selectedElement?.id]);
+
+  const isEditingMarkdown =
+    doubleClickedElement && isMdElement(selectedElement);
+
+  const editingFrameMetrics = children.props.componentProps?.isFreeMovement
+    ? {
+        left: children?.props?.componentProps?.positionX || 0,
+        top: children?.props?.componentProps?.positionY || 0,
+        width: children?.props?.width,
+        height: children?.props?.height
```

Comment:
Something is funky when playing around with this.  Here's what I see:

https://user-images.githubusercontent.com/430608/186027268-e426a1da-dd99-4b0e-8e1e-12a2ef89ea59.mp4

Reply:
Ah yes, In-line placement is busted and will be removed. That will be a follow up on this.